packages/common/src/visualizations/BarChartDataTransformer.ts

```diff
@@ -25,6 +25,31 @@ export class BarChartDataTransformer<TBarChartLayout, TPieChartConfig> {
 
         const DEFAULT_X_AXIS_TYPE = 'category';
 
+        const series = transformedData?.seriesColumns.map(
+            (seriesColumn, index) => {
+                const seriesLabel = Object.values(display?.series || {}).find(
+                    (s) => s.yAxisIndex === index,
+                )?.label;
+
+                return {
+                    dimensions: [
+                        transformedData.xAxisColumn.reference,
+                        seriesColumn,
+                    ],
+                    type: 'bar',
+                    name: seriesLabel || friendlyName(seriesColumn),
```

Comment:
@owlas I'm not sure what to do with this `seriesColumn` because when there's no label applied, we default to this which is like "Sum of Customer id", but on the Styling panel we don't have access to that `seriesColumn`, so we default to reference, which will be something like `customer_id` - what would you do here? 
Would you keep this `friendlyName(seriesColumn)` and keep the input empty if no override on the styling panel?